### ISSUE TITLE
P5 and 6S error message

### ISSUE THREAD

**Issue by nil2share** [xpdota/event-trigger#129]
Java SE 19 installed
triggevent-upd executed
ACt with the new overlay is updated

has been working for the last few weeks.

Appears to be throwing an error when entities like topaz stone spawn.  tried `master` and `combined-updates` branches

let me know if I need to gather any other info.

**Comment by xpdota** [comment]
Could you please attach triggevent.log in `%appdata%\triggevent` (or press advanced > open settings dir)?

**Comment by nil2share** [comment]
[triggevent.log](https://github.com/xpdota/event-trigger/files/9617437/triggevent.log)

**Comment by xpdota** [comment]
Hmm, looks like it might have to be exactly 17. Can you try the version with the bundled JRE (triggevent-win.zip)?

**Comment by nil2share** [comment]
yes, will go into p6s in a few hours, and will try that one and report back.  6 I believe I got the error when the + and x spawned.

**Comment by nil2share** [comment]
triggevent-win.zip seems to be fine. I will know for sure on tue when I do reclears of 5, 6 and 7 no errors with this one, but did get a few errors with the installed version

**Comment by xpdota** [comment]
In that case, I'm gonna assume it's just related to Groovy not supporting Java 18+ yet.
